docs/USER_RECORD.md

```diff
@@ -689,7 +689,7 @@ The following fields are defined in this section:
 hex. If any of the specified IDs match the system's local machine ID
 (i.e. matches `/etc/machine-id`) the fields in this object are honored.
 
-`matchHostname` → An array of string with a each a valid hostname. If any of
+`matchHostname` → An array of string with each a valid hostname. If any of
```

Comment:
This sentence reads a bit weird, I'm actually not sure what it's supposed to mean.

Reply:
"An array of strings, each one a valid hostname." ?

Reply:
Or maybe "An array of strings that are valid hostnames.".